pygame/_error.py

```diff
@@ -24,8 +24,13 @@ def unpack_rect(rect):
 
 
 def get_error():
-    return ffi.string(sdl.SDL_GetError())
+    err = ffi.string(sdl.SDL_GetError())
+    if not isinstance(err, str):
+        return err.decode('utf8')
```

Comment:
Does this return bytes in Python 2 and unicode in Python 3? Is that a good idea?